samples/secure-fraud-detection/README.md

```diff
@@ -0,0 +1,127 @@
+# Secure Fraud Detection Demo
+
+This demo showcases the implementation of a simple fraud detection system using LLMs (GPT-4 in this case) for users authenticated with Google.
+
+## The Demo
+
+### Setup
+
+The demo is based on fictional random data generated when the application starts, which includes:
+
+- 3 users
+- 50 transactions
+- For each transaction, a random amount between 1 and 1000 is generated and assigned to a random user. A random city is also assigned to each transaction.
+
+The setup is defined in the [Setup.java](./src/main/java/io/quarkiverse/langchain4j/samples/Setup.java) class.
+
+The users and transactions are stored in a PostgreSQL database. When running the demo in dev mode (recommended), the database is automatically created and populated.
+
+### Tools
+
+To enable fraud detection, we provide the LLM with access to customer and transaction data through two Panache repositories:
+
+- [CustomerRepository.java](./src/main/java/io/quarkiverse/langchain4j/samples/CustomerRepository.java)
+- [TransactionRepository.java](./src/main/java/io/quarkiverse/langchain4j/samples/TransactionRepository.java)
+
+### AI Service
+
+This demo leverages the AI service abstraction, with the interaction between the LLM and the application handled through the AIService interface.
+
+The interface uses specific annotations to define the LLM and the tools to be used:
```

Comment:
We should mention which interface we're talking about (`io.quarkiverse.langchain4j.sample.FraudDetectionAi`) and the snippets below look different from the actual code, so they need updating